pkg/webhook/package_webhook.go

```diff
@@ -57,6 +57,11 @@ func (v *packageValidator) Handle(ctx context.Context, request admission.Request
 			fmt.Errorf("decoding request: %w", err))
 	}
 
+	// Temporary fix since testing is currently broken
+	if p.Spec.Config == "" {
+		return admission.Allowed("Package contains empty configurations")
+	}
```

Comment:
We might have to settle for this for now, but this is going to allow empty configs for packages that have mandatory settings

Reply:
Yeah I wasn't able to test the other ideas since the tests are not working currently and the tests are the only way we can verify the webhook.